apps/src/lab2/views/ProjectBlockedUI.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+
+import AbuseExclamation from '@cdo/apps/code-studio/components/AbuseExclamation';
+import Alert, {alertTypes} from '@cdo/apps/componentLibrary/alert/Alert';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import i18n from '@cdo/locale';
+
+import Lab2Registry from '../Lab2Registry';
+
+import moduleStyles from './Lab2Wrapper.module.scss';
+
+export const ProjectBlockedUI: React.FunctionComponent<{
+  isProjectValidator: boolean;
+}> = ({isProjectValidator}) => {
+  const [showAlert, setShowAlert] = useState(true);
+  const projectManager = Lab2Registry.getInstance().getProjectManager();
+  const shareUrl = projectManager ? projectManager.getShareUrl() : null;
+  const isOwner = useAppSelector(state => state.lab.channel?.isOwner || false);
+
+  if (isProjectValidator) {
+    return (
+      <div
+        id="blocked-project-ui-container-project-validator"
+        className={moduleStyles.blockedProjectUIContainerProjectValidator}
+      >
+        {showAlert && (
+          <Alert
+            text={i18n.tosLong({url: 'http://code.org/tos'})}
```

Comment:
In the screenshot it looks like the alert component doesn't support markdown, should we make a version of the string without a link in it?

Reply:
Sure! Will do.